test/index.test.js

```diff
@@ -4,6 +4,11 @@
 const assert = require('chai').assert;
 const token = 'token';
 const testParseHook = require('./data/parseHookOutput.json');
+const readOnlyConfig = {
+    enabled: true,
+    username: 'headlessbot',
```

Comment:
in practice will this user end up same as `"sd-buildbot"` ? If so should we reuse the user refined in scm config section